ENIGMAsystem/SHELL/Graphics_Systems/Direct3D11/DX11blend.cpp

```diff
@@ -28,14 +29,13 @@ void update_blend_state() {
   static ID3D11BlendState* pBlendState = NULL;
   if (pBlendState) { pBlendState->Release(); pBlendState = NULL; }
   m_device->CreateBlendState(&blendStateDesc, &pBlendState);
-  draw_batch_flush(batch_flush_deferred);
+  enigma_user::draw_batch_flush(enigma_user::batch_flush_deferred);
```

Comment:
I can't figure out why `batch_flush_deferred` is in `enigma_user`. I'm not certain it belongs there (particularly under this name). It makes guarantees about the function of the underlying graphics system that I'm not sure we're ready to preserve. Anyway, guess we can deal with that, later.

Reply:
I discussed this with you over Discord briefly. I have seen a lot of users raising questions over how GMS "breaks" batches and sometimes running into issues from a batch not being "broke". We use the terminology "flush" instead of "break" since it's more common. Regardless, I exposed this API because I felt it would give us an edge against GMS because we can allow users to manually trigger a flush themselves, to change flushing modes, and better understand the flushing procedure in ENIGMA.